importer/tag/src/test/java/org/itsallcode/openfasttrace/importer/tag/TestTagImporter.java

```diff
@@ -380,4 +156,42 @@ private List<SpecificationItem> runImporter(final String content)
         factory.createImporter(file, builder).runImport();
         return builder.build();
     }
+
+    /**
+     * Create a test case using the content as input for the
+     * {@link TagImporter}. Make sure to concatenate the content to avoid
+     * breaking self-tracing.
+     * 
+     * @param content
+     *            content to parse
+     * @param itemBuilder
+     *            builder for the expected item
+     * @return the test case arguments
+     */
+    private static Arguments parsedItem(final String content, final SpecificationItem.Builder itemBuilder)
+    {
+        if (itemBuilder.build().getLocation() == null)
```

Comment:
I cant shake the feeling that the formatting is of:

`if`<space>`(`

and the JavaDoc look different than in the rest of the code.

Reply:
You are right, i fixed it.